spec/acceptance/game_nights_spec.rb

```diff
@@ -35,19 +35,21 @@
     header "Authorization", :auth_token
 
     example_request "POST /game_nights" do
-      expect(status).to eq 201
-
+      game_night = GameNight.first
       json_response = JSON.parse(response_body)
 
+      expect(status).to eq 201
+
       expect(json_response["data"]["attributes"]["name"]).
         to eq public_send(:name)
 
       expect(authenticated_user.game_nights.first.name).
         to eq public_send(:name)
 
-      game_night = GameNight.first
-
       expect(game_night.users.first).to eq authenticated_user
+
+      expect(authenticated_user.has_role? :organizer, game_night).
```

Comment:
Add parentheses to nested method call `authenticated_user.has_role? :organizer, game_night`.